src/projen/rest-api.ts

```diff
@@ -43,10 +48,10 @@ export class RestApi extends pj.Component {
         }
       }
     }
-    if (!fs.existsSync('./src/generated')) {
-      fs.mkdirSync('./src/generated');
+    if (!fs.existsSync(`${this.project.outdir}/src/generated`)) {
```

Comment:
Love it

Reply:
Oh, completely forgot to mention this! I stumbled over these hardcoded paths when writing unit tests. I hope this doesn't break something else!

Reply:
Will check

Reply:
Datastore and GraphQL aspects share the same relative paths, my gut says if it breaks something, I'd rather fix that than leave the old paths, even if currently unit testing was the first time I ran into problems with this.